pkg/controllers/replica-controller/handler.go

```diff
@@ -51,6 +51,10 @@ var (
 	upgradeMap = map[string]upgradeFunc{}
 )
 
+const (
+	restoreCompletedAnnotation = "openebs.io/restore-completed"
+)
+
```

Comment:
Yes. In relation to the above comment. This is a nice approach in my opinion
Although -- there is a proper home(openebs/api) for this -- but yes you can take into a next PR to move these buddies to their home. Your preference.

Reply:
ok, I see, that is another repo

Reply:
I put comment there